contrib/cirrus/runner.sh

```diff
@@ -118,6 +118,12 @@ function _run_endpoint() {
     make endpoint
 }
 
+function _run_minikube() {
+    _bail_if_test_can_be_skipped test/minikube
+    msg "Testing  minikube."
+    bats test/minikube/001-kube.bats |& logformatter
```

Comment:
Remove the `001`, just run bats on the entire directory. Unless you only ever gonna have `001.bats`

Reply:
fixed